docs/guides/base-images.mdx

```diff
@@ -7,6 +7,64 @@ description: "A guide to configuring a base image for your truss"
 Model serving enviroments will often be standardized as container images to avoid wrangling python, system, and other requirements needed to run your model on every deploy.
 Leverage your existing container artifacts by bringing your own base image to Truss.
 
+While the default image for the truss server in adequate for most use cases, there may come a time when you require a custom base image for your truss.
+For example, maybe the python packages required for you project are not compatible with the ones installed by default on Truss.
+In a situation like this, you can create your own base image based on the default truss image.
+
+## Creating a custom base image
```

Comment:
what do you think about having the `Setting a base image in config.yaml` section first, and then having this after that, with a transition like "If you don't have a base image ready yourself, and need to make one..."

Reply:
yeah, I think that format works as well